OpenRA.Mods.Common/Traits/World/PathFinder.cs

```diff
@@ -64,7 +64,10 @@ public PathFinder(World world)
 
 		public List<CPos> FindUnitPath(CPos source, CPos target, Actor self, Actor ignoreActor)
 		{
-			var li = self.Info.TraitInfo<MobileInfo>().LocomotorInfo;
+			var mobile = self.Trait<Mobile>();
+			var li = mobile.Info.LocomotorInfo;
```

Comment:
I'd either drop this (likely no real benefit) and use `locomotor.Info` directly, or at least move it below `locomotor` so you can just populate it by `locomotor.Info`.